packages/react-data-grid-react-window-grid/src/components/DataGridHeaderRow/useDataGridHeaderRow.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import {
+  ColumnIdContextProvider,
+  TableColumnDefinition,
+  useDataGridRow_unstable,
+} from '@fluentui/react-components';
+import type { CellRenderFunction } from '@fluentui/react-table';
+import {
+  DataGridHeaderRowProps,
+  DataGridHeaderRowState,
+} from './DataGridHeaderRow.types';
+/**
+ * Create the state required to render DataGridHeaderRow.
+ *
+ * @param props - props from this instance of DataGridHeaderRow
+ * @param ref - reference to root HTMLElement of DataGridHeaderRow
+ */
+export const useDataGridHeaderRow_unstable = (
```

Comment:
When navigating with keyboard the header scroll is not synchronized to scroll the body.

![header desync](https://github.com/microsoft/fluentui-contrib/assets/20744592/404b2407-b3d0-483b-acf4-f133333878d9)